### PR TITLE
updated layout size / reorganized props table look

### PR DESCRIPTION
**Pull request by fraincs**
- The props "table" is now responsive, clearer, and has a whole new layout, for better readibility.
- Updated the layout size to make the layout breathe more.

### CODE REVIEW

File: apps/docs/app/ui/components/propTable/PropTableRender.tsx [hunk 1]
```diff
@@ -16,32 +16,53 @@ export interface Item {
     type: ReactNode;
     defaultValue: string;
     description: ReactNode;
+    required: boolean;
 }
 
+const ColoredDefaultValue: React.FC<ColoredDefaultValueProps> = ({ defaultValue }) => {
+    const isBoolean = defaultValue === "true" || defaultValue === "false";
+    const formattedValue = isBoolean ? defaultValue.toString() : `'${defaultValue}'`;
```
Comment (vicky-comeau) [inline apps/docs/app/ui/components/propTable/PropTableRender.tsx:24]: couldn't you just do .toString and not check if it's a boolean?

File: apps/docs/app/ui/components/propTable/propTable.css [hunk 2]
```diff
@@ -12,73 +10,82 @@
...
+    padding: var(--hd-props-table-row-padding);
+}
+
+.hd-props-table__row:last-child {
+    border-bottom: 0;
 }
 
 .hd-props-table__cell {
-    display: var(--hd-props-table-cell-display);
+    display: flex;
```
[the remark this answers is not in the document]
Comment (fraincs) [reply apps/docs/app/ui/components/propTable/propTable.css:38]: True

File: apps/docs/app/ui/components/propTable/PropTableRender.tsx (hunk 1 above)
Comment (fraincs) [reply apps/docs/app/ui/components/propTable/PropTableRender.tsx:28]: On top of my head at this point everything is already a "string"

File: apps/docs/app/ui/components/propTable/propTable.css [hunk 3]
```diff
@@ -51,34 +39,52 @@
...
-    font-size: var(--hd-props-table-cell-description-font-size);
+.hd-props-table__name {
+    font-family: var(--hd-default-font-family);
+    font-size: 0.875rem;
+    font-weight: 600;
+    line-height: 1.5;
 }
 
+.hd-props-table__cell.hd-props-table__col-default {
+    font-size: 0.875rem;
```
Comment (vicky-comeau) [inline apps/docs/app/ui/components/propTable/propTable.css:80]: can we have a prop for these? There are many of the same font-sizes

Comment (fraincs) [reply apps/docs/app/ui/components/propTable/propTable.css:80]: Yes totally, well a variable ;p

Comment (vicky-comeau) [reply apps/docs/app/ui/components/propTable/propTable.css:80]: oops, yes variable xD my head is elsewhere I guess haha

**Review by vicky-comeau — APPROVED** [review]

**Pull request merged**